statics/js/components/topology.js

```diff
@@ -37,6 +35,10 @@ var TopologyComponent = {
               </i>\
             </span>\
           </button>\
+          <button id="expand" type="button" class="btn btn-primary" \
+                  title="Expand" @click="expandNextLevel">++</button>\
```

Comment:
I would prefer to use an awesome font like "fa-expand" and "fa-compress"

Reply:
yes, it will be awesome to have the icon but to 'expand the node tree' functionality, I am using those buttons. I can't use the same it will confuse the user :(